/*========================== begin_copyright_notice ============================

Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
See https://llvm.org/LICENSE.txt for license information.
SPDX-License-Identifier: Apache-2.0 with LLVM-exception

============================= end_copyright_notice ===========================*/

#    Description                  : Add possibility to use ChooseMSVCCRT-script, when
#                                   include LLVM library. 

--- a/llvm/cmake/modules/LLVMConfig.cmake.in
+++ b/llvm/cmake/modules/LLVMConfig.cmake.in
@@ -10,6 +10,11 @@ set(LLVM_PACKAGE_VERSION @PACKAGE_VERSION@)

 set(LLVM_BUILD_TYPE @CMAKE_BUILD_TYPE@)

+set(LLVM_USE_CRT_DEBUG @LLVM_USE_CRT_DEBUG@)
+set(LLVM_USE_CRT_MINSIZEREL @LLVM_USE_CRT_MINSIZEREL@)
+set(LLVM_USE_CRT_RELEASE @LLVM_USE_CRT_RELEASE@)
+set(LLVM_USE_CRT_RELWITHDEBINFO @LLVM_USE_CRT_RELWITHDEBINFO@)
+
 set(LLVM_COMMON_DEPENDS @LLVM_COMMON_DEPENDS@)

 set(LLVM_AVAILABLE_LIBS @LLVM_AVAILABLE_LIBS@)
